src/components/GameGrid/Rounds/KnockoutRounds.tsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react'
+
+import GameRound from './GameRound'
+import { Shout } from '../../../typography'
+
+import { makeStyles } from '@material-ui/styles'
+
+import { SinglePoll } from '../Graph'
+import TextContainer from '../../TextContainer'
+import Loud from '../../../typography/loud'
+
+const useStyles = makeStyles({
+  knockoutRoundsContainer: {
+    display: 'flex',
+    flexWrap: 'wrap',
+    gap: 8,
+    margin: 'auto',
+    alignItems: 'center',
+    justifyContent: 'center',
+  },
+  gameContainer: {
+    marginTop: 8,
+    width: 500,
+    maxWidth: '100%',
+  },
+  latestGame: {
+    padding: 16,
+    border: '2px solid #000',
+  },
+  latestGameContainer: {
+    marginTop: -16,
+    display: 'flex',
+    justifyContent: 'center',
+  },
+})
+
+interface KnockoutRoundData {
+  [key: number]: SinglePoll
+}
+
+interface Props {
+  knockoutRoundData: KnockoutRoundData
+}
+
+export default function KnockoutRounds(props: Props) {
+  const { knockoutRoundData } = props
```

Comment:
Overall, look great. Just some minor changes (mostly types).
 
Some major changes I'd like to suggest to the site before I give approval:
1. Move the Actibe Game above the KnockoutRound header
2. Make the active game graph a tad taller.

Other than that, look great, graphs are really nice and the styling on the active game is great.  TSX transition has also gone well (just please remember semi-colons)

Reply:
Have made type changes because I was unable to suggest them.